concepticondata/conceptlists/Morse-1996-375.tsv

```diff
@@ -0,0 +1,376 @@
+Morse-1996-375-198	198	curandero	medicine man		394	SHAMAN (FOLK HEALER)
+Morse-1996-375-199	199	cacique, curaca, capitán	chief		301	CHIEFTAIN
+Morse-1996-375-200	200	yo	I	*	1209	I
+Morse-1996-375-201	201	tú	thou	*	1215	THOU
+Morse-1996-375-202	202	él	he	*	1211	HE
+Morse-1996-375-203	203	ella	she		1834	SHE
+Morse-1996-375-204	204	ello	it		169	IT
+Morse-1996-375-205	205	nosotros	we	*	1212	WE
+Morse-1996-375-206	206	vosotros	ye	*	1213	YOU
+Morse-1996-375-207	207	ellos	they	*	817	THEY
+Morse-1996-375-208	208	mi mano	my hand			
+Morse-1996-375-209	209	tu mano	thy hand			
+Morse-1996-375-210	210	su mano	his hand			
+Morse-1996-375-211	211	nuestras manos	our hands			
+Morse-1996-375-212	212	vuestras manos	your hands			
+Morse-1996-375-213	213	sus manos de ellos	their hands			
+Morse-1996-375-214	214	mi arco	my bow			
+Morse-1996-375-215	215	tu arco	thy bow			
+Morse-1996-375-216	216	su arco – de él	his bow			
+Morse-1996-375-217	217	nuestro arco	our bow			
+Morse-1996-375-218	218	vuestro arco	your bow			
+Morse-1996-375-219	219	su arco – de ellos	their bow			
+Morse-1996-375-220	220	grande	big	*	1202	BIG
+Morse-1996-375-221	221	pequeño	small	*	1246	SMALL
+Morse-1996-375-222	222	frío	cold	*	1287	COLD
+Morse-1996-375-223	223	caliente	hot	*	1286	HOT
+Morse-1996-375-224	224	bueno	good	*	1035	GOOD
+Morse-1996-375-225	225	malo	bad	*	1292	BAD
+Morse-1996-375-226	226	blanco	white	*	1335	WHITE
+Morse-1996-375-227	227	negro	black	*	163	BLACK
+Morse-1996-375-228	228	vaya, ve	go!			
+Morse-1996-375-229	229	ven	come!			
+Morse-1996-375-230	230	come	eat!			
+Morse-1996-375-231	231	bebe!	drink!			
+Morse-1996-375-232	232	duerme!	sleep!			
+Morse-1996-375-233	233	corona	crown of head		1943	CROWN (OF HEAD)
+Morse-1996-375-234	234	dientes delanteros incisivos	front teeth		442	FRONT TOOTH (INCISOR)
+Morse-1996-375-235	235	punta de la lengua	tip tongue		443	TIP (OF TONGUE)
+Morse-1996-375-236	236	pelo largo	long hair		444	LONG HAIR
+Morse-1996-375-237	237	toda la garganta y cuello	around the neck			
+Morse-1996-375-238	238	nuez de la garganta	Adam’s apple		802	ADAM'S APPLE
+Morse-1996-375-239	239	espalda, parte superior	upper back		445	UPPER BACK
+Morse-1996-375-240	240	antebrazo	lower arm		446	LOWER ARM
+Morse-1996-375-241	241	muñeca	wrist		799	WRIST
+Morse-1996-375-242	242	la pierna inferior	lower leg		447	LOWER LEG
+Morse-1996-375-243	243	el pelo del cuerpo	body hair		189	HAIR (BODY)
+Morse-1996-375-244	244	estómago	stomach		819	STOMACH
+Morse-1996-375-245	245	intestinos	intestines	*	1991	INTESTINES
+Morse-1996-375-246	246	viejita	old woman		907	OLD WOMAN
+Morse-1996-375-247	247	nubes quietas	clouds at rest	*	1489	CLOUD
+Morse-1996-375-248	248	nubes de tormenta	storm clouds		448	STORM CLOUDS
+Morse-1996-375-249	249	chagra	corn field		449	CORN FIELD
+Morse-1996-375-250	250	quebrada	stream		947	STREAM
+Morse-1996-375-251	251	piedrecillas	pebbles		961	PEBBLES
+Morse-1996-375-252	252	rocas	huge rocks			
+Morse-1996-375-253	253	trocha	path		2252	PATH
```

Comment:
Is there a possible distinction between the two PATH mappings (IDs 253 and 85)?

Reply:
yes, this needs to be checked, please compare with what we did with Huber.

Reply:
Good catch. I will map 85 to 2457 PATH OR ROAD instead because they also have camino there which seems to be more standard road-y in meaning.